packages/components/psammead-copyright/README.md

```diff
@@ -0,0 +1,17 @@
+# Figure Copyright [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind=Copyright)
+
+### Description
+Displays the source attribution of an image in block capitals in the bottom-right of the parent element. 
+
+### When to use this component
+This component is intended to be used alongside images or diagrams contained within a [Figure component](../).
+
+### When not to use this component
+This component should not be used arbitrarily to represent source attribution across the page. As above, it is not intended to be used outside the [Figure component](../). The accessibility text would be incorrect and potentially confusing.
+
+Do not use this component if you know the source attribution is already covered by another copyright disclosure on the page, such as in the [Footer component](../../Footer).
+
+### Accessibility notes
+The default styling of this component is intended to comply with WCAG colour contrast standards.
+
+When used with screen-reading assistive technology, the component will read "Image source" before reading the attribution.
```

Comment:
Is this currently hard-coded to "Image source" or is it internationalisable? 

Should that live in a container rather than a presentational component? 🤔 

Not suggesting we refactor the code here, just document 👍

Reply:
This line is the readme is now outdated - the `Image source` has already been pulled out of this component and lives in the Copyright container. https://github.com/BBC-News/simorgh/blob/e66f40593fcf85436d421f25fb917c042caf7c73/src/app/containers/Copyright/index.jsx